packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -5,7 +5,7 @@ import * as path from 'path';
 
 const pinoLog =
   process.env.LOG_DESTINATION && process.env.LOG_DESTINATION !== 'console'
-    ? fs.createWriteStream(process.env.LOG_DESTINATION, {flags: 'w'})
+    ? fs.createWriteStream(process.env.LOG_DESTINATION, {flags: 'a'})
```

Comment:
Minor but it might be nice to drop a comment about what the `a` flag does

Reply:
https://github.com/statechannels/monorepo/pull/1475/commits/559e5a0cc1cf4d5ef84e5d517a418a76d6c66d9f